.eslintrc.json

```diff
@@ -15,11 +15,15 @@
       "no-console": 1,
       "indent": [ 1, 2 ],
       "comma-dangle": 0,
+      "object-curly-newline": "off",
       "curly": ["error", "multi-line"],
+      "max-len": "off",
```

Comment:
great job @urbanishimwe  however we talked about this in the meeting that they shouldn't be disabled `max-len": "off"` and `"no-await-in-loop": "off"`,